lib/manageiq/providers/amazon/engine.rb

```diff
@@ -1,4 +1,5 @@
-require 'rails/all'
```

Comment:
This was introduced in https://github.com/ManageIQ/manageiq-providers-amazon/pull/170/commits/69db2fcd84644b12125c2e680214e959bc323f77
The original and generated engine.rb does not require `rails/all`